lib/app/service/vendor.go

```diff
@@ -747,11 +771,12 @@ func resourcesFromPath(root string, includePatterns []string, ignorePatterns []s
 				return nil
 			}
 			log.Infof("Extracting images from Helm chart directory %v.", path)
-			resource, err := resourceFromChart(path)
+			resource, err := resourcesFromChart(path)
 			if err != nil {
-				return trace.Wrap(err)
+				return trace.Wrap(err).AddUserMessage("failed to parse Helm chart resources %q",
```

Comment:
here, saving a field would've been more appropriate:
```go
return trace.Wrap(err, "failed to parse Helm chart resources").AddField("path", path)
```

Reply:
Fields are not printed when the error is printed though (I don't think). Also, this is the error that users will see so I think it should be formatted properly and fields are more suitable for logs than for user-facing messages.

Reply:
Fields are only dumped with `DebugReport` yes.